packages/lit-html/src/lit-html.ts

```diff
@@ -1044,9 +1092,13 @@ export class NodePart {
       // itemParts always have end nodes
       this._clear(itemPart?._endNode!.nextSibling);
     }
+    return hasDisconnect;
   }
 
   private _clear(start: ChildNode | null = this._startNode.nextSibling) {
+    if (this._hasDisconnect) {
+      this._disconnectValue();
```

Comment:
Why is this `_disconnectValue` rather than `_disconnect`? Maybe add a comment to clarify?

Reply:
Done